lib/wagn/application.rb

```diff
@@ -9,9 +9,10 @@
   # Bundler.require(:default, :assets, Rails.env)
 end
 
-module ActiveSupport::BufferedLogger::Severity
-  WAGN = UNKNOWN + 1
-  
+module ActiveSupport::Logger::Severity
+#  WAGN = UNKNOWN + 1
+  WAGN = 6
```

Comment:
yeah, though I haven't really played with it yet and I wouldn't be surprised if I just broke it.

Philipp put this together as a way of logging wagn-specific performance details.

(His solution is better than mine, but "UNKNOWN" was breaking for some reason.